.gitignore

```diff
@@ -62,4 +62,14 @@ typings/
 
 # Generated files and folders
 packages/client/build/**/*
-log.txt
\ No newline at end of file
+log.txt
+
+# Ignore local and special envs
+
+# loaded in all cases
```

Comment:
@lilitkarapetyan - if we're ignoring these environment files, they're not present in the repo. That means they're not available to developers, to get them started.

Or, am I missing something?

Reply:
All default settings must be saved in the ".env" file and must be overridden by special envs (.env.local, .env.development.local ...)

For example, for development, we need to create .env.development and put all the dev configs inside, and if some developer wants to change the settings only for his computer, he can add the .env.development.local (gitignored) file and override the settings. env.development, and there is no need to edit .env files before each pull/push request

Reply:
Here's the priority of the files for the development build and the production build:

Dev.: (npm start): .env.development.local, .env.development, .env.local, .env

Prod.: (npm run build): .env.production.local, .env.production, .env.local, .env

If you ever want to use something in your local environment without being specific to the development build or the production build, you can add some variables to your .env.local file.

Reply:
local means specific configs for current server or workstation so it shouldn't be deployed to git. It can be also content server database passwords (just example not for our case)

Reply:
Just checked we are using .env.local as main env file it need to be renamed as .env
![изображение](https://user-images.githubusercontent.com/20753599/69011411-8aaa3e00-0983-11ea-96fd-9ec7b5244e1c.png)

Reply:
That's great @lilitkarapetyan - thanks for explaining.  

(sorry if I forget what you've taught me, probably in a few days :-( )